src/app/containers/RadioPageBlocks/Blocks/OnDemandEpisodeImage/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import Image, { AmpImg } from '@bbc/psammead-image';
+import {
+  GEL_SPACING_QUAD,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_1_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/breakpoints';
+import { RequestContext } from '#contexts/RequestContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const ImageContainer = styled.div`
+  padding: ${GEL_SPACING_QUAD}
+    ${({ dir }) => (dir === 'ltr' ? GEL_SPACING_DBL : 0)} ${GEL_SPACING_QUAD}
```

Comment:
I'm not sure this is necessary. I think this sort of thing is handled automatically. Sareh might be able to provide more insight

Reply:
This might be different because this padding is image specific. From the [zeplin](https://app.zeplin.io/project/5d4d77b1fcbfc79bf47e0294/screen/5df769324d5dda1bce94f519) prototypes, the image has a direction specific padding, but that could be interpreted differently by UX.